packages/shared/lib/core/profile/actions/profiles/checkAndMigrateProfiles.ts

```diff
@@ -289,3 +291,11 @@ function persistedProfileMigrationToV13(
 
     saveProfile(newProfile as IPersistedProfile)
 }
+
+function persistedProfileMigrationToV14(existingProfile: IPersistedProfile): void {
+    const isLedgerProfile = existingProfile?.type === ProfileType.Ledger
+    if (isLedgerProfile) {
+        existingProfile.strongholdVersion = undefined
```

Comment:
maybe you can try a delete so the key also gets removed?
```suggestion
        delete existingProfile.strongholdVersion
```